yarn-project/circuits.js/src/structs/private_circuit_public_inputs.ts

```diff
@@ -219,6 +226,7 @@ export class PrivateCircuitPublicInputs {
       fields.callContext,
       fields.argsHash,
       fields.returnValues,
+      new Fr(fields.metaHwm),
```

Comment:
why does this need the Fr constructor? Why not just `fields.metaHwm`?

Reply:
Oups, left over. I originally had `metaHwm` down as just a `number`. Fixed now